src/Path/UploadListings.jsx

```diff
@@ -128,23 +131,60 @@ function UploadListings() {
 
   const [localImages, setLocalImages] = useState([]);
   const handleMultipleInputChange = (event) => {
-    const newImages = Array.from(event.target.files);
-    setLocalImages((prevImages) => [...prevImages, ...newImages]);
-    setImage([...image, ...newImages]);
+    const newFiles = Array.from(event.target.files);
+
+    if (image.length > 0) {
+      const validImages = newFiles.filter(file => file.type.startsWith("image/"));
```

Comment:
Here, instead of checking for valid images. check for invalid images. If the length > 0, throw an error. Suppose a user uploads both image and pdf, you will simply filter out the pdf s currently. Throw an error so that it is understood, else clients will again come back telling pdf is not being uploaded

Reply:
Done